src/humanize/locale/ru_RU/LC_MESSAGES/humanize.po

```diff
@@ -248,17 +248,17 @@ msgstr "девять"
 #, python-format
 msgid "%d microsecond"
 msgid_plural "%d microseconds"
-msgstr[0] ""
-msgstr[1] ""
-msgstr[2] ""
+msgstr[0] "микросекунда"
+msgstr[1] "микросекунды"
+msgstr[2] "микросекунд"
```

Comment:
The `%d` will be replaced by a number. So in English:

"%d microsecond" -> "1 microsecond"
"%d microseconds" -> "2 microseconds"

Is this correct?

```suggestion
msgstr[0] "%d микросекунда"
msgstr[1] "%d микросекунды"
msgstr[2] "%d микросекунд"
```